src/NServiceBus.RavenDB.TransactionalSession.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus.TransactionalSession
+{
+    public sealed class RavenDbOpenSessionOptions : NServiceBus.TransactionalSession.OpenSessionOptions
+    {
+        public RavenDbOpenSessionOptions(System.Collections.Generic.IDictionary<string, string> multiTenantConnectionContext = null) { }
+    }
+    public static class RavenDbTransactionalSessionExtensions
```

Comment:
Do we want to add `Open` extension method to support scenarios with no options object passed?

Reply:
No this has been discussed already. We said for persisters that have optional arguments, we don't do that because the options object is the one that needs to be passed.